<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Core::Internal::SettingsDialog</class>
 <widget class="QDialog" name="Core::Internal::SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="pageTree">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <column>
       <property name="text">
        <string>0</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout">
       <item>
        <widget class="QStackedWidget" name="stackedPages">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>250</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Core::Internal::SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Core::Internal::SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
